upup/pkg/fi/cloudup/awsup/machine_types.go

```diff
@@ -605,6 +605,51 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
 		EphemeralDisks: []int{320, 320},
 	},
 
+	// x1e family
+
+	{
```

Comment:
Looks like you've transposed MemoryGB and Cores :)

Reply:
Fixed, thanks. Ugh.